### ISSUE TITLE
scrolling through layers with mousewheel

### ISSUE THREAD

**Issue by sebasdfian** [xournalpp/xournalpp#4438]
**Is your feature request related to a problem? Please describe.**
I sometimes have to copy&paste notes on different layers between several open instances of xournal. I am significantly quicker in this task when using the mouse. 
When scrolling from page to page in the original file, xournal automatically switches to the layer last edited on the new page (which might be useful for other reasons).
I now have to switch to the layer I want to copy from with two mouseclicks on the "layer" option. (First world problems :-) )

**Describe the solution you'd like**
It would be significantly quicker if xournal allowed to change layers with the mousewheel while hovering the pointer over the "layer counter". 
This behaviour is already implemented with the "page counter", so there might be some code available to reuse?

As long as there are not too many layers, a +/-button like with the "page counter" would not make switching between layers in a tablet&pen-scenario significantly more difficult. 
Or the drop-down-menu for changing layers could be kept in addition to the "layer counter"/mousewheel. This would probably not add too much bulk to the interface [see mock-up](https://ibb.co/tsspVT3)

**Describe alternatives you've considered**
The "problem" could also be solved by making the selected layer a document-wide option. However, this will probably break someone elses workflow (TM) and might be problematic in the case of different numbers of layers on different pages.

Thank you very much for your consideration. 

Sebastian

**Comment by rolandlo** [comment]
Have you considered writing a [plugin](https://xournalpp.github.io/guide/plugins/plugins/) that switches to the layer you want? Note that `app.setCurrentLayer` can be used to switch layers. 
https://github.com/xournalpp/xournalpp/blob/0612bf5fbdb8bbb63a43e20ea9b35223b8bbbc0c/src/core/plugin/luapi_application.h#L1735-L1746

In your mock-up layer names are ignored, which doesn't look ideal.

**Comment by bhennion** [comment]
I don't think this is really suitable for a plugin. I think the asked behaviour is somewhat natural and should be part of the main app.
